modules/ROOT/pages/asset-details.adoc

```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
 This is similar to using a program such as Postman to specify custom HTTP headers and test each API's method using the API's endpoint.
 . To connect the API to one of your applications and use the API to send and receive data, click *Request access*.
 
+== View Conformance Status 
+
+You can view governance conformance status in Exchange for versions of API definitions. The status is one of the following:
+ 
+* A conformance badge is displayed for a selected version of an API.
```

Comment:
while the above sentence says the status can be "one of the following", the text following that is not enumerating the possible status values such as [conformant, not conformant, not validated] etc.. The possible status values need to be documented here with an explanation as to what the value means and in what cases a certain value is present